charlib/cli/utils.py

```
@@ -14,22 +14,46 @@ def find_yaml_files(path) -> list:
         return []
 
 
+def resolve_subkey(value):
+    """If a config value ends in .yml or .yaml, resolve it to the YAML contents."""
+    if isinstance(value, str):
+        if not any(value.lower().endswith(ext) for ext in ['.yml', '.yaml']):
+            return value
+        possible_yamls = find_yaml_files(value)
+        if len(possible_yamls) != 1:
+            raise ValueError(f'Unable to resolve {value} to a unique existing file')
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)